duckduckgo/BrowserServicesKit
# Automatically add the PR link to the Asana task

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/1199333091098016/1204360503924262/f
iOS PR: N/A
macOS PR: N/A
What kind of version bump will this require?: No release required, this is a workflow change only

**Optional**:

CC: @ayoy 

**Description**:

This PR adds a GitHub Actions workflow to automatically add a PR link to the referenced Asana task.



**Steps to test this PR**:
1. Quick approach: check the Asana task and verify that a comment was posted
1. More in depth approach: create a branch off of this branch, make a public Asana task, then open a PR which references it and validate that the comment appears

Also, we want to test that draft PRs don't trigger this workflow. You can see [this PR](https://github.com/duckduckgo/BrowserServicesKit/pull/386) which includes a comment to the skipped CI run, or you can create a branch from this PR and open a new draft PR with it to be sure.



**OS Testing**:

* N/A

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
